### ISSUE TITLE
Update datahub configuration

### ISSUE THREAD

**Issue by pierrejego** [georchestra/ansible#111]
Datahub need to get configuration and assets from datadir to custom

**Comment by landryb** [comment]
technically it's a bit gross, since if one modifies the config in the datadir, it wont be used, and ppl will get puzzled ...

locally on my wip datahub deployment i have this in my nginx config:
```
                location ~ /datahub/assets/img/(.*).(webp|png|svg) {
                        alias /etc/georchestra/datahub/assets/img/$1.$2;
                }
                location = /datahub/assets/favicon.ico {
                        alias /etc/georchestra/datahub/assets/favicon.ico;
                }
                location = /datahub/assets/configuration/default.toml {
                        alias /etc/georchestra/datahub/assets/configuration/default.toml;
                }
```

**Comment by landryb** [comment]
will by closed by #127 which has the above config

**Comment by landryb** [comment]
closed by #127
